pyega3/pyega3.py

```diff
@@ -114,15 +120,15 @@ def get_token(credentials):
     r = requests.post(URL_AUTH, headers=headers, data=data)
 
     try:
-        print('')
+        logging.info('')
```

Comment:
Why we want a blank INFO line here? :-) This will be looking like:
`16:07:26 INFO   `

Reply:
yes it is for adding a blank line after printing header info, makes it better to view logs